src/site-server.ts

```diff
@@ -132,6 +133,11 @@ export class SiteServer {
 			await startProxyServer();
 		}
 
+		let filteredBlueprint = this.meta.blueprint?.blueprint;
+		if ( filteredBlueprint ) {
+			filteredBlueprint = filterUnsupportedFeatures( filteredBlueprint );
+		}
```

Comment:
NIT maybe we could make the `filterUnsupportedFeatures` to expect `undefined`, so using it would be more simple:
```suggestion
		const filteredBlueprint = filterUnsupportedFeatures( this.meta.blueprint?.blueprint );
```